packages/sources/eth-beacon/src/config/index.ts

```diff
@@ -1,54 +1,32 @@
-import { Requester, util } from '@chainlink/ea-bootstrap'
-import { Config } from '@chainlink/ea-bootstrap'
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
 
-export const NAME = 'ETH_BEACON'
-
-export const DEFAULT_ENDPOINT = 'balance'
-export const ENV_ETH_CONSENSUS_RPC_URL = 'ETH_CONSENSUS_RPC_URL'
-export const ENV_ETH_EXECUTION_RPC_URL = 'ETH_EXECUTION_RPC_URL'
-export const ENV_FALLBACK_ETH_CONSENSUS_RPC_URLS = ['RPC_URL', 'ETHEREUM_RPC_URL']
-export const ENV_BATCH_SIZE = 'BATCH_SIZE'
-export const ENV_GROUP_SIZE = 'GROUP_SIZE'
-export const ENV_CHAIN_ID = 'CHAIN_ID'
-
-export const DEFAULT_BATCH_SIZE = 15
-export const DEFAULT_GROUP_SIZE = 15
-export const DEFAULT_CHAIN_ID = 1
-
-export type EthBeaconConfig = Config & {
-  adapterSpecificParams: {
-    beaconRpcUrl: string
-    executionRpcUrl: string
-    batchSize: number
-    groupSize: number
-    chainId: number
-  }
-}
-
-export const makeConfig = (prefix?: string): EthBeaconConfig => {
-  const beaconRpcUrl = util.getRequiredEnvWithFallback(
-    ENV_ETH_CONSENSUS_RPC_URL,
-    ENV_FALLBACK_ETH_CONSENSUS_RPC_URLS,
-    prefix,
-  )
-  const executionRpcUrl = util.getEnv(ENV_ETH_EXECUTION_RPC_URL) || ''
-  const batchSize = Number(util.getEnv(ENV_BATCH_SIZE, prefix) ?? DEFAULT_BATCH_SIZE)
-  const groupSize = Number(util.getEnv(ENV_GROUP_SIZE, prefix)) || DEFAULT_GROUP_SIZE
-  const chainId = Number(util.getEnv(ENV_CHAIN_ID, prefix) || DEFAULT_CHAIN_ID)
-  const defaultConfig = Requester.getDefaultConfig(prefix)
-  return {
-    ...defaultConfig,
-    api: {
-      ...defaultConfig.api,
-      baseURL: beaconRpcUrl,
-    },
-    defaultEndpoint: DEFAULT_ENDPOINT,
-    adapterSpecificParams: {
-      beaconRpcUrl,
-      executionRpcUrl,
-      batchSize,
-      groupSize,
-      chainId,
-    },
-  }
-}
+export const config = new AdapterConfig({
+  ETH_CONSENSUS_RPC_URL: {
+    description: 'RPC URL of an Ethereum consensus client (beacon node)',
+    type: 'string',
+    required: true,
+  },
+  ETH_EXECUTION_RPC_URL: {
+    description:
+      'RPC URL of an Ethereum execution client (archive node). Required for requests that need a limbo validator search',
+    type: 'string',
+    default: '',
```

Comment:
I think we can remove this default. It's ok if the value for the env var is undefined. We only had the `|| ''` in the v2 adapter to force the type for `adapterSpecificParams`. Luckily in v3 we don't have to anymore

Reply:
Removing the default causes type issues in proof of reserves EA. balance.ts file imports eth-beacon EA